src/Aspire.Hosting.Garnet/GarnetContainerImageTags.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Garnet;
+
+internal static class GarnetContainerImageTags
+{
+    public const string Registry = "docker.io";
+    public const string Image = "ghcr.io/microsoft/garnet";
+    public const string Tag = "1.0.2";
```

Comment:
Isn't this what we want?

```suggestion
    public const string Registry = "ghcr.io";
    public const string Image = "microsoft/garnet";
    public const string Tag = "1.0.2";
```

Reply:
Absolutely right. Thanks!